### PR TITLE
Feat/solana support

### PR DESCRIPTION
**Pull request by jazg**

### CODE REVIEW

File: compat/v0/compat_test.go [hunk 1]
```diff
@@ -137,30 +137,28 @@ var _ = Describe("Compat V0", func() {
...
 		Expect(storedHash).Should(Equal(hash))
 
 		// btc txhash mapping
 		keys, err = client.Keys("*").Result()
 
 		// v0 hash should have a mapping in the store
-		Expect(keys).Should(ContainElement("npiRyatJm8KSgbwA/EqdvFclMjfsnfrVY2HkjhElEDk="))
-
-		// v1 hash should be correct
-		Expect(hash).Should(Equal("YlkYzfTTCcptfS4bYdxnrXXNMv-C_6Y1UzWwi_wOrGI"))
```
Comment (TheAncientGoat) [inline compat/v0/compat_test.go:163]: Any chance we can maintain a v1 hash correctness check?

File: compat/v0/compat_test.go [hunk 2]
```diff
@@ -171,30 +169,28 @@ var _ = Describe("Compat V0", func() {
...
 		Expect(storedHash).Should(Equal(hash))
 
 		// btc txhash mapping
 		keys, err = client.Keys("*").Result()
 
 		// v0 hash should have a mapping in the store
-		Expect(keys).Should(ContainElement("Q1E14yjJGkz6Oe5VPIK3vX/A7q93qKF6Hof6DGQ/yW4="))
-
-		// v1 hash should be correct
-		Expect(hash).Should(Equal("y4ol_nr7P9IXsW59AiAkIuUW5ytMKkwcEVihNZlPvJQ"))
```
Comment (TheAncientGoat) [inline compat/v0/compat_test.go:197]: Same here

**Review by TheAncientGoat — APPROVED** [review]
A minor comment, but looks good otherwise

File: compat/v0/compat_test.go (hunk 1 above)
Comment (jazg) [reply compat/v0/compat_test.go:163]: Yep, added.

**Review by TheAncientGoat — APPROVED** [review]

**Pull request merged**